configure-ironic.sh

```diff
@@ -19,7 +19,11 @@ if [[ $IRONIC_FAST_TRACK == true ]]; then
     INSPECTOR_POWER_OFF=false
     # TODO(dtantsur): ipa-api-url should be populated by ironic itself, but
     # it's not yet, so working around here.
-    INSPECTOR_EXTRA_ARGS=" ipa-api-url=http://${IRONIC_URL_HOST}:6385"
+    if [ ! -z "$CACERT_FILE" ] || [ ! -z "$INSECURE" ]; then
```

Comment:
Is the 'OR' condition because we foresee a configuration where we would have the ca-cert file and yet we would not enable TLS? or should we use 'AND', or rather just check the existense of ca-cert to enable TLS?

Reply:
even when INSECURE is false the value is set right? is this condition correct?

Reply:
I was thinking we could have a configuration without CA cert, but the user wants to use insecure to not verify the cert with the CA, hence the "or". However, as pointed out by @dtantsur we should probably not set the CA cert at all, and not allow for insecure.